# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../orbitcpp/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur orbitcpp-1.3.9-orig/orbitcpp/idl-compiler/error.h orbitcpp-1.3.9/orbitcpp/idl-compiler/error.h
--- orbitcpp-1.3.9-orig/orbitcpp/idl-compiler/error.h	2008-07-23 04:17:49.000000000 +0000
+++ orbitcpp-1.3.9/orbitcpp/idl-compiler/error.h	2008-07-23 04:19:56.000000000 +0000
@@ -35,7 +35,7 @@
 #include <libIDL/IDL.h>
 #include <stdexcept>
 #include <string>
-
+#include <cstring>
 
 
 
